core/main/src/firebolt/rpc.rs

```diff
@@ -72,3 +86,58 @@ where
     }
     rpc_module
 }
+
```

Comment:
`OpenRPCSchemaRegistrar` receives requests from extensions to register any openrpc schemas so that ripple processes them just as done for the base firebolt schema.